android/app/src/main/java/org/chrome/device/ml/ml/FeatureConverter.java

```diff
@@ -0,0 +1,105 @@
+/* Copyright 2020. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.chrome.device.ml.ml;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.chrome.device.ml.tokenization.FullTokenizer;
+
+/**
+ * Convert String to features that can be fed into BERT model.
+ */
+public final class FeatureConverter {
```

Comment:
would be good to add a unit test for this

Reply:
third party library.